pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -62,19 +61,23 @@ func NewDevWorkspaceReconciler() *DevWorkspaceReconciler {
 }
 
 func (d *DevWorkspaceReconciler) Reconcile(ctx *chetypes.DeployContext) (reconcile.Result, bool, error) {
-	if isDevWorkspaceOperatorCSVExists(ctx) {
-		// Do nothing if DevWorkspace has been already deployed via OLM
-		return reconcile.Result{}, true, nil
-	}
-
 	if infrastructure.IsOpenShift() {
-		wtoInstalled, err := isWebTerminalSubscriptionExist(ctx)
-		if err != nil {
```

Comment:
So, we do not care about WTO anymore from Che / DevSpaces operator pov? It is actually interesting that on Developer Sandbox we had both DWO and WTO installed in the `crw` namespace  so looks like both isDevWorkspaceOperatorCSVExists and isWebTerminalSubscriptionExist returned false

Reply:
We don't need WTO check anymore, since if WTO installed then DWO is installed as well. And we already have improved DWO check. 

`isWebTerminalSubscriptionExist` returned false because it checked `openshift-operators` namespace for existed WTO subscription but not `crw` namespace.